contracts/Minter.sol

```diff
@@ -0,0 +1,183 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.11;
+
+interface IMintable {
+  function mint(address to, uint256 amount) external;
+  function decimals() external returns (uint256);
+}
+
+/// @title A Minter contract for Splinterlands
+/// @author Splinterlands Team (@fbslo)
+
+contract SPSMinter {
+  /// @notice Address that can change pools
+  address public admin;
+  /// @notice Address of the token to mint
+  IMintable public token;
```

Comment:
`token` can be immutable to save some gas

Reply:
Resolved: https://github.com/steem-monsters/sps-minter/commit/95869a888ae3e8b7c1632d3781445d24fb217ae9